package.json

```diff
@@ -48,11 +52,19 @@
     ]
   },
   "devDependencies": {
-    "ava": "^0.11.0",
-    "xo": "^0.12.1"
+    "ava": "^0.22.0",
+    "eslint": "^4.8.0",
+    "eslint-config-standard-with-typescript": "^1.0.1",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^5.2.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^3.0.1",
```

Comment:
https://github.com/standard/eslint-config-standard/issues/72

Reply:
This is the reason we have these plugins as `peerDepenencies`. I think `eslint-config-standard` should normal depend on them but this is the situation currently.